packages/ui/src/1_atoms/Paragraph/Paragraph.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+
+import React from 'react';
+
+import { Paragraph } from './Paragraph';
+
+describe('Test paragraphs', () => {
```

Comment:
```suggestion
describe('Paragraph', () => {
```
I'd probably just use a component name in `describe` because if you run `yarn test`, you know that you see the output from tests so `test` is unnecessary word and if we use something different than just a component name, it will clutter the log. What do you think? cc @soulBit

Reply:
Agreed, "Test" is extraneous

Reply:
ok, will change